File: app/src/main/java/com/example/client/screens/reset/present/IPasswordResetPresent.java
package com.example.client.screens.reset.present;

import android.widget.TextView;

public interface IPasswordResetPresent {
    void onVertification(String email, String code);
    void onResetPassword(String email, String password);
    void onSendEmail(String email);
    void onCountDownTimer(TextView tv,Integer time);
}

File: app/src/main/java/com/example/client/screens/reset/present/IPasswordResetPresent.kt
package com.example.client.screens.reset.present;

import android.widget.TextView;

public interface IPasswordResetPresent {
    void verification(String email, String code);
    void resetPass(String email, String password);
    void sendRequest(String email);
    void onCountDownTimer(TextView tv,Integer time);
}
